docs/user/YAL5-codon-pairs_usage.md

```diff
@@ -0,0 +1,53 @@
+# Running YAL5-codon-pairs.R
+
+## Input ##
+The script `YAL5-codon-pairs.R` looks into metafeatures, currently the translational landscape surrounding codon pairs. 
+
+- h5 file
+- gff3 file
+- tsv codon positions file
+
+Read counts are fetched from the H5 file of a sample and assigns the counts to the cooresponding codon pair position for the genes using the species-specific GFF3 file and a tsv file containing the codon positions for all genes of the species being studied, i.e. the `yeast_codon_table.tsv`. The script calculates the normalised number of reads within a window surrounding each occurrence of the feature of interest (e.g. a codon pair) which are then overlayed to generate a single plot. Normalisation is carried out within a set window of codons adjacent to the feature of interest. The default value of this window is currently 5 codons either side of the feature of interest. 
+The aim of this script is to study translation by examining whether specific features can slow down translation, thereby having an inhibitory effect. If the feature is being translated more slowly than its surrounding codons this will generate a peak. 
```

Comment:
Maybe move this to the top as it is a summary of the script and not about the inputs. 

Potentially reword
`by examining whether specific features can slow down translation` -> `by creating a meta feature plot showing the the region surrounding a feature of interest. This can be used to investigate whether specific features can slow down translation`